### ISSUE TITLE
你好，这个2d的结果是正常的吗？ 为什么越变越低

### ISSUE THREAD

**Issue by Be997398715** [Hsankesara/VoxelMorph-PyTorch#4]
![image](https://user-images.githubusercontent.com/40139193/93057265-36094980-f6a0-11ea-82e8-001b4965aa27.png)
用的您自带的数据集

**Comment by hktxt** [comment]
same problem.
I think it's worng!
loss cant be negtive!

**Comment by domadaaaa** [comment]
@Be997398715 Hello, same problem, did you solved it?

> [ 0.29 mins] After 1 epochs, the Average training loss is  tensor(-0.7222, device='cuda:0') and average DICE score is tensor(0.4526)
[ 0.07 mins] After 1 epochs, the Average validations loss is  tensor(-0.5995, device='cuda:0') and average DICE score is tensor(0.4311)
[ 0.26 mins] After 2 epochs, the Average training loss is  tensor(0.8095, device='cuda:0') and average DICE score is tensor(0.1617)
[ 0.07 mins] After 2 epochs, the Average validations loss is  tensor(-0.0179, device='cuda:0') and average DICE score is tensor(0.0002)
[ 0.26 mins] After 3 epochs, the Average training loss is  tensor(-0.0010, device='cuda:0') and average DICE score is tensor(0.0007)
[ 0.07 mins] After 3 epochs, the Average validations loss is  tensor(-0.0055, device='cuda:0') and average DICE score is tensor(0.0014)
[ 0.26 mins] After 4 epochs, the Average training loss is  tensor(-0.0016, device='cuda:0') and average DICE score is tensor(0.0015)
[ 0.07 mins] After 4 epochs, the Average validations loss is  tensor(-0.0057, device='cuda:0') and average DICE score is tensor(0.0018)
[ 0.27 mins] After 5 epochs, the Average training loss is  tensor(-0.0012, device='cuda:0') and average DICE score is tensor(0.0017)
[ 0.07 mins] After 5 epochs, the Average validations loss is  tensor(-0.0056, device='cuda:0') and average DICE score is tensor(0.0019)
